preact.dev.js

```diff
@@ -41,7 +44,7 @@ export class PathLookup extends Component {
     const path = location.hash ? transformHash(location.hash): defaultPath;
     this.setState({
       params: null,
-      path,
+      ...path,
       current: null
     })
     this.hashChange(path);
```

Comment:
keep in mind that RouterOutlet extends from PathLookup and if you call hashChange with an string then u will run into a TypeError